src/test/shell/integration/rc_options_test.sh

```diff
@@ -96,4 +96,19 @@ function test_options_override() {
   expect_not_log '^xxxxxxxxxxx'
 }
 
+function test_bep_option_source() {
+  local -r build_event_text_file="$(pwd)/bep.txt"
+  local -r pkg=$FUNCNAME
+  create_pkg $pkg
+  bazel build --build_event_text_file="$TEST_log" \
+      $pkg:xxxxxxxxxxxxxxxxxxxxxxxxxtrue || fail "build failed"
+  if is_windows; then
+    # source is printed as PathFragment, which uses `/` instead of `\`.
+    expected_path=$(echo "$bazelrc" | sed 's/^C:/c:/; s/\//\\\\/g')
```

Comment:
```suggestion
    expected_path=$(echo "$bazelrc" | sed 's|^C:|c:|; s|/|\\\\\\\\|g')
```

Reply:
For the doubling of `\` see the new comment above. I tested that this works on Windows.

I also replaced `/` as sed delimiter with `|` to make the `sed` expression less confusing to read (too many `/` and `\` otherwise).

Reply:
Still too many `/` `|` `\` to be super readable tho 😅

Thanks for figuring this one out